pynest/examples/Potjans_2014/README.rst

```diff
@@ -1,61 +1,108 @@
+The command for running the script with two MPI processes would be:
+
+.. code-block:: bash
+
+   mpirun -n 2 python run_microcircuit.py
+
+Note on Parameters
+##################
+
+By default, the simulation uses external Poissonian input to excite all neuronal populations of the microcircuit, see ``poisson_input': True`` in ``network_params.py``.
+If set to ``False``, the Poissonian input is turned off and compensated approximately by calculated DC input.
+In addition to this ongoing external drive, a thalamic stimulation or a stimulation by an external DC input can be switched on in ``stimulus_params.py`` (the default for both types of stimuli is ``False``).
+
+The default random initialization of membrane voltages in this simulation uses population-specific means and standard deviations to reduce an initial activity burst in the network: ``'V_type': 'optimized'`` in ``network_params.py``.
+Previous implementations used the same mean and standard deviation for all populations which corresponds to setting ``'V_type': 'original'``.
+
+Contributions to this PyNEST Microcircuit
+#########################################
+
+Current communicating author: Johanna Senk
+
+2016: first version implemented by Hendrik Rothe, Hannah Bos and Sacha van Albada
+
+2019: optimized initialization of membrane voltages added by Han-Jia Jiang
+
+2020: revision of code and documentation, and adaptation to NEST3 by Johanna Senk
+
+Citation
+########
+
+If you use this code, we ask you to cite the paper by Potjans and Diesmann [1]_ and the NEST release on Zenodo.
+
+Acknowledgments
+###############
+
+This project has received funding from the European Union Seventh Framework Programme ([FP7/2007-2013]) under grant agreement n° 604102 (Human Brain Project, HBP), and the European Union’s Horizon 2020 Framework Programme for Research and Innovation under Specific Grant Agreement No. 720270 (Human Brain Project SGA1) and No. 785907 (Human Brain Project SGA2).
+
+Funding for [1]_: This work was supported by the Helmholtz Alliance on Systems Biology; European Union (FACETS, grant 15879 and BrainScaleS, grant 269921); Deutsch-Israelische Projektkooperation (DIP, grant F1.2); Bundesministerium für Bildung und Forschung, Germany (BMBF, grant 01GQ0420 to BCCN Freiburg), and the Next-Generation Supercomputer Project of the Ministry of education, culture, sports, science and technology (MEXT), Japan. Funding to pay the Open Access publication charges for this article was provided by Research Center Juelich a member of the Helmholtz Association.
+
+Other Implementations of the Microcircuit Model
+###############################################
+
+The original `SLI version <https://github.com/nest/nest-simulator/tree/master/examples/nest/Potjans_2014>`__ by Tobias Potjans and Markus Diesmann (later revised by David Dahmen and Sacha van Albada) is also part of the NEST code base as an example.
```

Comment:
`The original SLI version by Tobias Potjans and Markus Diesmann ([INSERT YEAR]) is included in the NEST code base as an example. This version was revised by David Dahmen and Sacha van Albada in [INSERT YEAR]`.

Reply:
The version by Tobias was never included in NEST. Also, Tom helped create the SLI version that is part of NEST. Maybe reword to 'A SLI version by David Dahmen, Tom Tetzlaff, and Sacha van Albada, based on the original version by Tobias Potjans and Markus Diesmann, is also part of the NEST code base as an example.'

Reply:
I used the version by @albada without the years as they are not precisely known.
@albada: Can we also name authors of the PyNN version?